src/Components/Components/src/RenderTree/RenderTreeDiffBuilder.cs

```diff
@@ -31,7 +29,7 @@ enum DiffAction { Match, Insert, Delete }
             var editsBufferStartLength = editsBuffer.Count;
 
             var diffContext = new DiffContext(renderer, batchBuilder, componentId, oldTree.Array, newTree.Array);
-            AppendDiffEntriesForRange(ref diffContext, 0, oldTree.Count, 0, newTree.Count);
+            AppendDiffEntriesForRange(renderer, ref diffContext, 0, oldTree.Count, 0, newTree.Count);
```

Comment:
`Renderer` is already a property on the `diffContext` so doesn't need to be passed around separately.